### ISSUE TITLE
TypeScript definition file

### ISSUE THREAD

**Issue by Bnaya** [bramstein/fontfaceobserver#64]
Will you accept typescript definition file to this repo?

Thanks!

**Comment by fela98** [comment]
+1
I would really appreciate a TypeScript definition file

**Comment by bramstein** [comment]
Sure! That sounds like a good idea. Pull request welcome 👍

**Comment by renestalder** [comment]
Someone used this and can explain how this works? because it has no default export.

**Comment by drewsilcock** [comment]
Including the JavaScript file adds `window.FontFaceObserver`, which allows you to do `new FontFaceObserver(fontName)` without importing anything. The Typings file just tells TypeScript that FontFaceObserver is a global thing and basically not to worry about it, allowing you to do `new FontFaceObserver(fontName)` in TypeScript and get those yummy typings and autocompletions and intellisense. This is why you pass the `--global` flag in when you do the `typings install`.

tldr; you don't need to import anything, you can just use it.

**Comment by renestalder** [comment]
Yeah well I see, that was my fault. I have some kind of babel/typescript stuff in my build that makes it possible to import JavaScript files when a specific export standard of another module system is applied. So I then can import some libraries directly instead of loading it globally, which doesn't work in that case. But of course It does when I add it globally. Thanks for your feedback.
